internal/runner/runner.go

```diff
@@ -114,11 +114,11 @@ func (r *CommandRunner) runStep(ctx context.Context, step *CommandStep, dry bool
 	if err != nil {
 		return nil, err
 	}
-	posixEnv = append(posixEnv, "PWD="+pwd)
+	// posixEnv = append(posixEnv, "PWD="+pwd)
 
 	cmd := exec.CommandContext(ctx, name, args...)
 	cmd.Dir = pwd
-	cmd.Env = posixEnv
+	// cmd.Env = posixEnv
```

Comment:
You need to set `cmd.Env` otherwise `exec.Command` adds `PWD` and [copies the current process' env vars](https://cs.opensource.google/go/go/+/refs/tags/go1.18.8:src/os/exec/exec.go;l=1162-1191;drc=3a41094107237535673bdbbdaabc10e54cb2b67c). It's hard to know what those env vars are as the logic changes a bit from version to version of Go.

```suggestion
	cmd.Env = posixEnv
```